marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java

```diff
@@ -256,6 +257,10 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
 
     public void setHubConfig(HubConfig hubConfig) {
         this.hubConfig = hubConfig;
+        if (this.entityManager == null) {
```

Comment:
Would this ever happen? I don't think it's necessary, as long as this command class is assumed to be instantiated via Spring. I don't think just instantiating EntityManagerImpl and then setting hubConfig on it will often work, as EntityManagerImpl has other dependencies (HubProject, could be more in the future) that aren't being set. 

I thinking asserting that entityManager is not null may be worthwhile, although a user couldn't do anything with the error - it would reflect a system misconfiguration (Spring has a handy Assert class for such purposes).

Reply:
Note that on develop, there's now an EntityManagerImpl(HubConfig) constructor. I recommend using that because it also assigns a value to the HubProject field.